README.md

```diff
@@ -248,6 +248,21 @@ Hasura is an open source GraphQL engine that deploys instant, realtime GraphQL A
 - [Tutorial: Fullstack React Native with GraphQL and Authentication](https://blog.hasura.io/tutorial-fullstack-react-native-with-graphql-and-authentication-18183d13373a/)
 - [Guide: Hasura GraphQL Engine One-click App on DigitalOcean Marketplace](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/digital-ocean-one-click.html)
 
+## Developer Productivity [↗](https://awesomestacks.dev/developer-productivity)
+
+Boost your Productivty with GitHub, Jira, Jenkins, Grafana and Prometheus.
+
+- [GitHub](https://github.com/) - [🛠](https://stackshare.io/github) - The site to keep your code and open it to world.
+- [Jira](https://es.atlassian.com/software/jira) - [🛠️](https://stackshare.io/jira) - Agile software development.
+- [Jenkins](https://jenkins.io/) - [🛠️](https://stackshare.io/jenkins) - [🐙](https://github.com/jenkinsci/jenkins) - Open source automation server for CI/CD.
```

Comment:
Let's have Jenkins, Grafana and Prometheus be the first tools since they are the key ones. GitHub and Jira could go afterwards then.